src/diffusers/loaders/lora.py

```diff
@@ -292,81 +224,74 @@ def lora_state_dict(
 
         return state_dict, network_alphas
 
-    @classmethod
-    def _best_guess_weight_name(
-        cls, pretrained_model_name_or_path_or_dict, file_extension=".safetensors", local_files_only=False
+    def fuse_lora(
```

Comment:
Does this method only exist to map `fuse_unet` to `fuse_denoiser`? If yes, this could be added as a comment. Same for other similar methods.

Reply:
Done.

Reply:
don't think this method is needed, we can just map the argument from the method in `LoraBaseMixin`
we should have this class for loading only and `LoraBaseMixin` for the other methods

Reply:
@yiyixuxu I hear your point but I feel relatively strongly about having this method to act as an interface (as explained in the comments). 

Otherwise, the `fuse_lora()` method would need to have `fuse_unet`, `fuse_transformer` arguments and we may have to support other arguments based on the kind of diffusion backbone newly introduced in the literature (a mamba, for example).

With the current implementation, we have a nice logical segregation of the concepts through the `fuse_denoiser` argument. The `denoiser` here can be any model architecture in practice as long as there's compatibility and support. 

Furthermore, when the users would call `fuse_lora()` on an SD3 pipeline they would see `fuse_unet` in the docstrings or on their IDEs, too. This can be confusing for them. 

So, keeping all of these in mind, I would prefer the current implementation.

Reply:
we can start to deprecate `fuse_unet`, `fuse_tranformer`  etc and only use `fuse_denoiser`from this point on, no?
when these fuse_unet`, `fuse_tranformer` are passed in `**kwargs`, we can map it to `fuse_denoiser`

Reply:
Yes that makes perfect sense to me. Will do that :)

Reply:
b404a3278 should have taken care of it.